src/vs/editor/common/editorCommon.ts

```diff
@@ -4164,6 +4167,33 @@ export var EventType = {
 };
 
 /**
+ * Positions on the view
+ */
+export const ViewPosition = {
+	LineStart: 'lineStart',
+	LineFirstNonWhitespaceCharacter: 'lineFirstNonWhitespaceCharacter',
+	LineCenter: 'lineCenter',
+	LineEnd: 'lineEnd',
+	LineLastNonWhitespaceCharacter: 'lineLastNonWhitespaceCharacter'
+};
+
+/**
+ * @internal
+ */
+export var CommandDescription= {
+	CursorMove: <ICommandHandlerDescription>{
+		description: nls.localize('editorCommand.cursorMove.description', "Move cursor to a logical position in the view"),
+		args: [
+			{
```

Comment:
@jrieken Is this correct? I think the name and description apply to `firstArgument.to` not to `firstArgument`.